### PR TITLE
[FIX] List page spacing 

### PR DESCRIPTION
**Pull request by nerdlet**
## Description

#180935241

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Screenshots
Screenshot 2022-01-20 at 16 40 05

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: src/components/InvestigationsCard/useStyles.js [hunk 1]
```diff
@@ -28,10 +28,16 @@ const useStyles = makeStyles(({ breakpoints, typography }) => ({
...
+    "& span": {
+      boxShadow: `0 4px 8px 0 rgba(0,0,0,0.4)`,
+    },
   },
   featured: {
     height: `${typography.pxToRem(421)} !important`,
     width: `${typography.pxToRem(300)} !important`,
+    "& span": {
+      boxShadow: `0 4px 8px 0 rgba(0,0,0,0.4)`,
+    },
```
Comment (gertie-sheshe) [inline src/components/InvestigationsCard/useStyles.js:40]: Are the boxShadow changes meant to be on this PR? @nerdlet 😅

Comment (nerdlet) [reply src/components/InvestigationsCard/useStyles.js:40]: @gertie-sheshe Yikes, pushed this in the wrong branch. i'll revert it

File: src/components/Lists/useStyles.js [hunk 2]
```diff
@@ -10,6 +10,9 @@ const useStyles = makeStyles(({ typography, palette, breakpoints }) => ({
       justifyContent: "space-between",
     },
   },
+  list: {
+    paddingBottom: typography.pxToRem(48),
```
Comment (gertie-sheshe) [inline src/components/Lists/useStyles.js:14]: Seems to be 80px from the designs?

Screenshot 2022-01-19 at 23 19 30

File: src/components/Lists/index.js [hunk 3]
```diff
@@ -98,7 +98,9 @@ function Lists({ results: listsProp, paginationProps, ...props }) {
   return (
     <div className={classes.root}>
       <div className={classes.section}>
-        <Typography variant="h2">Your Lists</Typography>
+        <Typography variant="h2" className={classes.list}>
```
Comment (kilemensi) [inline src/components/Lists/index.js:101]:
```suggestion
        <Typography variant="h2" className={classes.title}>
```

File: src/components/Lists/index.js [hunk 4]
```diff
@@ -98,7 +98,9 @@ function Lists({ results: listsProp, paginationProps, ...props }) {
...
+          Your Lists
```
Comment (kilemensi) [inline src/components/Lists/index.js:102]: Add a story in PT to move all of these hard-coded strings to props & CMS.

**Review by kilemensi — COMMENTED** [review]
👍🏽 

--

Still can't run TrollTracker locally so don't wait for my approval on this. My only question is why are you adding spaces in two places; padding-bottom on title and margin-top on create button? On of these should be enough since both of these should be below the title.

**Review by KhadijaMahanga — COMMENTED** [review]
Looks Good!

I agree with @kilemensi on the `padding-bottom` & `margin-top` on two subsequent element. 

Also for smaller screen the `Create New List` does not align with the download actions
Screenshot 2022-01-20 at 12 44 38

File: src/components/ContentActions/index.js [hunk 5]
```diff
@@ -56,7 +56,7 @@ function ContentActions({ apiUri, queryParams, type, ...props }) {
     <div className={classes.root}>
       <Section className={classes.section}>
         <Grid container justifyContent="space-between" alignItems="center">
-          <Grid container item alignItems="center">
+          <Grid container item xs={4} alignItems="center">
```
Comment (kilemensi) [inline src/components/ContentActions/index.js:59]:
```suggestion
          <Grid container item md={4} alignItems="center">
```
1. We _want_ this to take the whole width in small screens,
2. Looks like you're changing this part of the code from 2 different PRs; this and #69 [here](https://github.com/CodeForAfrica/TwoopsTrackerUI/pull/69/files#diff-2071c343700538b0f64649995af238b5baf56a9e79e88e505f5de65c5b6b0d02L58-R84). My advice would be make this change in one of the PRs.

**Review by kilemensi — COMMENTED** [review]
👍🏽 

---

For everything under desktop (`md` and down), remove the left padding on file Download.

S

**Comment by kilemensi** [comment]
merge conflicts @nerdlet

File: src/components/AccountsList/useStyles.js [hunk 6]
```diff
@@ -2,6 +2,20 @@ import { makeStyles } from "@material-ui/core/styles";
 
 const useStyles = makeStyles(({ typography, breakpoints }) => ({
   root: {},
+  section: {
+    margin: `${typography.pxToRem(20)} 0`,
+    display: "flex",
+    justifyContent: "space-between",
+    padding: 0,
+  },
```
Comment (kilemensi) [inline src/components/AccountsList/useStyles.js:10]: Aren't we supposed to add spacing to around `title`? Why are we adding it to `section`?

**Review by kilemensi — COMMENTED** [review]
👍🏽

File: src/components/AccountsList/useStyles.js (hunk 6 above)
Comment (nerdlet) [reply src/components/AccountsList/useStyles.js]: @kilemensi I think it works without any spacing both on the title or section. Removed i
Screenshot 2022-01-28 at 15 51 44
t 👇🏽

**Review by kilemensi — COMMENTED** [review]
🚀

File: src/components/ContentActions/index.js [hunk 7]
```diff
@@ -14,6 +14,7 @@ const useStyles = makeStyles(({ palette, typography }) => ({
   section: {
     borderBottom: "1px solid #00000040",
     paddingBottom: typography.pxToRem(28),
+    padding: 0,
```
Comment (KhadijaMahanga) [inline src/components/ContentActions/index.js:17]: @nerdlet not sure why we are adding `padding zero` here, and there is `paddingButton`

**Review by KhadijaMahanga — APPROVED** [review]
🚀

File: package.json [hunk 8]
```diff
@@ -110,7 +110,7 @@
     ]
   },
   "engines": {
-    "node": ">=12.18.0 <15.x",
+    "node": ">=12.18.0 <17.x",
```
[the remark this answers is not in the document]
Comment (nerdlet) [reply package.json:113]: @kilemensi Hahaha, blame genderGap 😞

[the remark this answers is not in the document]
Comment (kilemensi) [reply package.json:113]: May be you should use [`nvm`](https://github.com/nvm-sh/nvm)?

File: src/components/ContentActions/index.js (hunk 7 above)
Comment (nerdlet) [reply src/components/ContentActions/index.js:17]: @KhadijaMahanga Um, I think I removed this...let me confirm.

File: package.json (hunk 8 above)
[the remark this answers is not in the document]
Comment (nerdlet) [reply package.json:113]: > May be you should use [`nvm`](https://github.com/nvm-sh/nvm)?

@kilemensi  💃🏽 This is definitely the tool I need at the moment. Thanks!

File: src/components/ContentActions/index.js (hunk 7 above)
Comment (nerdlet) [reply src/components/ContentActions/index.js:17]: @KhadijaMahanga Yap removed it.

**Pull request merged**